src/igvfd/audit/whole_organism.py

```diff
@@ -0,0 +1,30 @@
+from snovault.auditor import (
+    audit_checker,
+    AuditFailure,
+)
+from .formatter import (
+    audit_link,
+    path_to_text,
+)
+
+
+@audit_checker('WholeOrganism', frame='object')
+def audit_whole_organism_human_taxa(value, system):
+    '''Flag whole organisms associated with taxa Homo sapiens.'''
+    if 'donors' in value:
+        donor_ids = value.get('donors')
+        taxa_dict = {}
+        for d in donor_ids:
+            donor_object = system.get('request').embed(d + '@@object?skip_calculated=true')
```

Comment:
if we are not using the donor that was human or not in the audit detail - why bother with dictionary? I think you can get away with set of the taxas?, and probably need to do the traversal - ONLY if the "taxa" calculated property is not in the sample?

Reply:
Got it, code is adjusted to use 'taxa' first if present. If not, donors are traversed and a set() of taxas is checked